### PR TITLE
Rhiros 579 add report date

### PR DESCRIPTION
**Pull request by PreetiW**
### Description
* Added new column `Last reported` in the systems UI and report
* fixed default sorting based on `Last reported` instead of name

**Jira:** 
https://issues.redhat.com/browse/RHIROS-579
https://issues.redhat.com/browse/RHIROS-194

**References**
https://momentjs.com/
https://momentjscom.readthedocs.io/en/latest/moment/04-displaying/02-fromnow/

### Screenshots

image

image

image

### CODE REVIEW

File: src/Components/Reports/Util.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
 import { get } from 'lodash';
+import moment from 'moment';
 import { pdfRowKeys, percentageKeys, reportRowKeys, SYSTEMS_REPORT_FILE_NAME } from '../../constants';
 
+export const formatLastReported = (date) => {
+    const today = moment().format('YYYY-MM-DD');
+    const isTodayWithoutTime =  today === date;
+    return isTodayWithoutTime ? 'Today' : moment(date).fromNow();
```
Comment (PreetiW) [inline src/Components/Reports/Util.js:8]: Reference for using this mothod: https://momentjscom.readthedocs.io/en/latest/moment/04-displaying/02-fromnow/

Comment (r14chandra) [inline src/Components/Reports/Util.js:8]: If a system is updated/uploaded today then it's showing `Today` not as `Just now` or `X minutes ago`.

Screenshot from 2022-04-11 20-19-46

**Review by r14chandra — CHANGES_REQUESTED** [review]

File: src/Components/Reports/Util.js (hunk 1 above)
Comment (PreetiW) [reply src/Components/Reports/Util.js:8]: Hello @r14chandra that's expected as we have only date information coming from the backend for `report_date` field, once the backend will send time information as well this code will handle that. Please refer to below test cases for same.

Comment (PreetiW) [reply src/Components/Reports/Util.js:8]: Hello @r14chandra that's expected as we have only date information coming from the backend for `report_date` field, once the backend will send time information as well this code will handle that. Please refer to below test cases for same.

**Review by r14chandra — APPROVED** [review]
LGTM! 👍🏻

File: src/Components/Reports/Util.js (hunk 1 above)
Comment (r14chandra) [reply src/Components/Reports/Util.js:8]: Got it.

**Review by kgaikwad — COMMENTED** [review]
@PreetiW - could you please do rebase for this branch?

**Review by kgaikwad — APPROVED** [review]

**Comment by PreetiW** [comment]
Thank you @kgaikwad and @r14chandra for the review, merging this!

**Pull request merged**